src/webui/api/transfercontroller.h

```diff
@@ -37,6 +37,14 @@ class TransferController : public APIController
 
 public:
     using APIController::APIController;
+    bool isActionSafe(const QString &actionName) const override
+    {
+        return m_safeActions.contains(actionName);
```

Comment:
please move function definition to .cpp file.

Reply:
I think there's a case to be made for keeping the definition in the header file. The functions are required to be implemented in every controller and only expose a private variable that isn't otherwise referenced. Truthfully, they're also an ugly hack to add metadata to each function. I'd prefer if they remained in the header files, but I don't have super strong feelings about it.